src/Mod/PartDesign/Gui/TaskExtrudeParameters.cpp

```diff
@@ -681,16 +681,18 @@ void TaskExtrudeParameters::getReferenceAxis(App::DocumentObject*& obj, std::vec
     }
 }
 
-void TaskExtrudeParameters::onButtonFace(const bool pressed)
+void TaskExtrudeParameters::onButtonFace(const bool checked)
 {
+    if (!checked && ui->lineFaceName->text().isEmpty())
+        ui->lineFaceName->setPlaceholderText(tr("No face selected"));
+    else if (checked && ui->lineFaceName->text().isEmpty())
+        ui->lineFaceName->setPlaceholderText(tr("Click on a face in the model"));

```

Comment:
This text should be as small as possible. Alternatively, maybe move this to this to the bottom info bar?
![Screenshot from 2022-07-17 19-35-28](https://user-images.githubusercontent.com/4316933/179402093-117ae398-d132-4ddb-a77b-62ee71da5bad.png)

Reply:
That depends strongly on the font the user uses, the screen resolution and the width he uses for the task panels.

The user can widen the task panel if he needs to read everything. but you only need to read it once, then you get the idea how the workflow is. So in your screenshot is OK in my opinion.

Concerning the width, when you e.g. add a PD hole you will also have to widens your task panel to see everything and there it is even mandatory otherwise you cannot access all edits.

Reply:
This one is fine, but if you can add a tooltip to it that might be good. As such this should be handled in Qt itself I suppose.  I haven't worked with hole, but if it is as described it's a design flaw and it must be changed.